service/service_test.go

```diff
@@ -63,7 +63,7 @@ func (s *ServiceSuite) BeforeTest(suiteName, testName string) {
 
 func (s *ServiceSuite) Test_AddUseCase() {
 	ucMock := mocks.NewUseCaseInterface(s.T())
-	ucMock.EXPECT().AddFeatures().Return().Once()
+	ucMock.EXPECT().AddFeatures().Return(nil).Once()
 	ucMock.EXPECT().AddUseCase().Return().Once()
 
 	s.sut.AddUseCase(ucMock)
```

Comment:
Please check for the error return value 

```
	err := s.sut.AddUseCase(ucMock)
	assert.Nil(s.T(), err)
```

and add handling of the error case to this test, e.g. via:

```
func (s *ServiceSuite) Test_AddUseCase_Error() {
	ucMock := mocks.NewUseCaseInterface(s.T())

	expErr := errors.New("error")
	ucMock.EXPECT().AddFeatures().Return(expErr).Once()

	err := s.sut.AddUseCase(ucMock)
	assert.NotNil(s.T(), err)
	assert.Equal(s.T(), expErr, err)
}
```